packages/tasit-action/src/ethers.test.js

```diff
@@ -93,4 +74,34 @@ describe("ethers.js", () => {
       newValue,
     ]);
   });
+
+  describe("message signing", () => {
+    const rand = Math.floor(Math.random() * Math.floor(1000)).toString();
+    let rawTx, signedTx;
+
+    beforeEach("", async () => {
+      const data = contract.interface.functions.setValue.encode([rand]);
+
+      rawTx = {
+        gasLimit: 64000,
+        to: contract.address,
+        data: data,
+        nonce: await provider.getTransactionCount(wallet.address),
+      };
+    });
+
+    it("should sign set contract's value tx", async () => {
+      signedTx = await wallet.sign(rawTx);
+      expect(signedTx).to.exist;
+    });
+
+    it("should set contract's value using signed tx", async () => {
+      const sentTx = await provider.sendTransaction(signedTx);
```

Comment:
I don't love it that the test uses a var from a previous test. Maybe wrap the setup into a beforeEach for a sub-describe here?

Reply:
Moved to https://github.com/tasitlabs/TasitSDK/issues/106

Reply:
Moved to https://github.com/tasitlabs/TasitSDK/issues/106

Reply:
Okay awesome - I'm resolving this then.